src/js/jsonEslint.js

```diff
@@ -30,81 +30,72 @@ fluid.defaults("fluid.lintAll.jsonEslint", {
+
+        try {
+            var validationPromise = eslint.lintText(wrappedContent);
+            eslintPromises.push(validationPromise);
+            validationPromise.then(function (results) {
+                fluid.each(results, function (singleResult) {
+                    singleResult.filePath = pathToFile;
+                    eslintResults.push(singleResult);
                 });
-            } catch (err) {
-                fluid.fail(err);
-            }
-
-            that.results.checked++;
-        });
-
-        var eslintSequence = fluid.promise.sequence(eslintPromises);
-        eslintSequence.then(
-            function () {
-                fluid.each(eslintResults, function (singleError) {
-                    var relativePath = path.relative(that.options.rootPath, singleError.filePath);
-
-                    var fileErrorCount    = fluid.get(singleError, "errorCount");
-                    if (fileErrorCount) {
-                        that.results.invalid++;
-                        that.results.errorsByPath[relativePath] = [];
-
-                    }
-                    else {
-                        that.results.valid++;
-                    }
-
-                    fluid.each(singleError.messages, function (singleMessage) {
-                        // Our "wrapper" adds three lines of code before the first line.
-                        var adjustedLine = singleMessage.line - 3;
-                        // Our "wrapper" adds 17 characters to the first line.
-                        var adjustedColumn = adjustedLine === 1 ? singleMessage.column - 17 : singleMessage.column;
-
-                        that.results.errorsByPath[relativePath].push({
-                            line: adjustedLine,
-                            column: adjustedColumn,
-                            message: singleMessage.message
-                        });
+            });
+        } catch (err) {
+            fluid.fail(err);
+        }
+    });
+
+    var eslintSequence = fluid.promise.sequence(eslintPromises);
+    eslintSequence.then(
+        function () {
+            fluid.each(eslintResults, function (singleError) {
+                var relativePath = path.relative(that.options.rootPath, singleError.filePath);
+
+                var fileErrorCount    = fluid.get(singleError, "errorCount");
+                if (fileErrorCount) {
+                    that.results.invalid++;
+                    that.results.errorsByPath[relativePath] = [];
+
+                }
+                else {
+                    that.results.valid++;
+                }
+
+                fluid.each(singleError.messages, function (singleMessage) {
+                    // Our "wrapper" adds three lines of code before the first line.
+                    var adjustedLine = singleMessage.line - 3;
+                    // Our "wrapper" adds 17 characters to the first line.
```

Comment:
What are the 17 characters?

Reply:
I'll update the comment to refer people to the named variable.